### PR TITLE
fix: Make event-invoices endpoint admin only

### LINKED ISSUE #7063: event-invoices endpoint failing
/event-invoices endpoint is failing because it has `has_organizer` decorator set up which either expects `event_id` param in URL or expects user to be an admin.

This is obviously not the case with the endpoint. Hence, it should be fixed

### PR DESCRIPTION
**Pull request by Haider8**
Fixes #7063 

#### Short description of what this resolves:
Currently `/event-invoices` endpoint requires the `event_id` param in url or expects the user to be admin, which is not the case for this endpoint.

#### Changes proposed in this pull request:
Remove decorators `is_organizer` and `is_admin`.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ app/api/event_invoices.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5776766&bid=18908204)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ app/api/event_invoices.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5776766&bid=18908204)

**Comment by Haider8** [comment]
@iamareebjamal Please review

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/api/event_invoices.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5776766&bid=18908204)

File: app/api/helpers/permission_manager.py [hunk 1]
```diff
@@ -61,8 +61,9 @@ def is_organizer(view, view_args, view_kwargs, *args, **kwargs):
     if user.is_staff:
         return view(*view_args, **view_kwargs)
 
-    if user.is_owner(kwargs['event_id']) or user.is_organizer(kwargs['event_id']):
-        return view(*view_args, **view_kwargs)
+    if kwargs.get('event_id'):
```
Comment (iamareebjamal) [inline app/api/helpers/permission_manager.py:64]: This can be extracted in a variable and reused

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/api/event_invoices.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5776766&bid=18908204)

**Comment by iamareebjamal** [comment]
Ignore remaining deepsource issues. We should only fix code related to issue. Don't revert the already made changes though

**Pull request merged**